docs/retrieve.txt

```diff
@@ -0,0 +1,456 @@
+
+      Use the following syntax to specify the query:
+
+      .. code-block:: go
+
+         $movies = DB::connection('mongodb')
+               ->collection('movies')
+               ->where('imdb.votes', '>', 1300000)
+               ->get();
+
+   .. tab:: Run the Query
+      :tabid: query-run
+
+      To run the find operation and view the results, edit the ``show()`` function
+      in the ``MovieController.php`` file to resemble the following code:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: php
+
+            class MovieController
+            {
+               public function show()
+               {
+                  $movies = DB::connection('mongodb')
+                        ->collection('movies')
+                        ->where('imdb.votes', '>', 1300000)
+                        ->get();
+
+                  return view('browse_movies', [
+                     'movies' => $movies
+                  ]);
+               }
+            }
+
+         .. output::
+            :language: none
+            :visible: false
+
+            Title: The Shawshank Redemption
+            Year: 1994
+            Runtime: 142
+            IMDB Rating: 9.3
+            IMDB Votes: 1513145
+            Plot: Two imprisoned men bond over a number of years, finding solace and eventual redemption
+            through acts of common decency.
+
+            Title: The Dark Knight
+            Year: 2008
+            Runtime: 152
+            IMDB Rating: 9
+            IMDB Votes: 1495351
+            Plot: When the menace known as the Joker wreaks havoc and chaos on the people of Gotham, the
+            caped crusader must come to terms with one of the greatest psychological tests of his ability to
+            fight injustice.
+
+      Then, make the following changes to your Laravel Quick Start application:
+
+      - Add the ``use Illuminate\Support\Facades\DB`` use statement to your ``MovieController.php``
+        file
+      - Replace the contents of your ``browse_movies.blade.php`` file with the following code:
+
+        .. code-block:: php
+
+           <!DOCTYPE html>
+           <html>
+           <head>
+              <title>Browse Movies</title>
+           </head>
+           <body>
+           <h2>Movies</h2>
+
+           @forelse ($movies as $movie)
+           <p>
+              Title: {{ $movie['title'] }}<br>
+              Year: {{ $movie['year'] }}<br>
+              Runtime: {{ $movie['runtime'] }}<br>
+              IMDB Rating: {{ $movie['imdb']['rating'] }}<br>
+              IMDB Votes: {{ $movie['imdb']['votes'] }}<br>
+              Plot: {{ $movie['plot'] }}<br>
+           </p>
+           @empty
+              <p>No results</p>
+           @endforelse
+
+           </body>
+           </html>
+
+        .. note::
+
+           This change is necessary because you cannot use the ``->`` object operator to access document fields when running
```

Comment:
Suggestion:

I think it could be better if the admonition identified which change it refers to.
E.g.

"Since the Laravel query builder returns ... the view accesses the fields by using the ... syntax instead of the "->" object operator."